main.css

```diff
@@ -7,6 +7,79 @@ html, body
     flex-direction: column;
 }
 
+footer
+{
+    position: fixed;
+    bottom: 0;
+    right: 0;
+    width: 100%;
+    height: 20px;
+    z-index: 9999;
+    background-color: transparent;
+}
+
+#notifications
+{
+    position: fixed;
+    top: 45px;
+    right: 0px;
+    height: auto;
+    width: 25%;
```

Comment:
Width `50%` makes sure this covers the gap in the center, and it gives us lots of room to display the error cleanly. Should be `100%` on mobile probably